code/game/gamemodes/nations/nations.dm

```diff
@@ -88,15 +97,27 @@ datum/game_mode/nations
 				H << "You are now part of the great sovereign nation of [H.mind.nation.name]!"
 				continue
 
+			if(H.mind.assigned_role in civilian_positions)
+				H << "You do not belong to any nation and are free to sell your services to the highest bidder."
+				continue
+
 			else
 				message_admins("[H.name] with [H.mind.assigned_role] could not find any nation to assign!")
 				continue
 
 /datum/game_mode/nations/proc/set_ai()
 	for(var/mob/living/silicon/ai/AI in mob_list)
-		AI.laws = new (pick(subtypesof(/datum/ai_laws/nations)))
+		AI.set_zeroth_law("")
+		AI.clear_supplied_laws()
+		AI.clear_ion_laws()
+		AI.clear_inherent_laws()
+		AI.add_inherent_law("Uphold the Space Geneva Convention: Weapons of Mass Destruction and Biological Weapons are not allowed.")
+		AI.add_inherent_law("You are only capable of protecting crew if they are visible on cameras. Nations that willfully destroy your cameras lose your protection.")
+		AI.add_inherent_law("Subdue and detain crewmembers who use lethal force against each other. Kill crew members who use lethal force against you or your borgs.")
```

Comment:
It most likely will be at a certain point down the line, once the language is locked down and we have a fancy name for it and what now. Since everything is still in heavy flux regarding AI/borg involvement, I'm holding off on that bit until a further patch